src/main/java/teammates/ui/controller/AdminEmailLogPageAction.java

```diff
@@ -115,10 +115,12 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         }
         nextEndTimeToSearch = query.getEndTime();
         
-        String status = "&nbsp;&nbsp;Total Logs gone through in last search: " + totalLogsSearched + "<br>";
+        String status = new StringBuilder().append("&nbsp;&nbsp;Total Logs gone through in last search: ")
+                                           .append(totalLogsSearched).append("<br>")
         //link for Next button, will fetch older logs
-        status += "<button class=\"btn-link\" id=\"button_older\" onclick=\"submitFormAjax('"
-                  + nextEndTimeToSearch + "');\">Search More</button>";
+                                           .append("<button class=\"btn-link\" id=\"button_older\" onclick=\"submitFormAjax('")
+                                           .append(nextEndTimeToSearch).append("');\">Search More</button>")
+                                           .toString();
```

Comment:
This can be done the old-fashioned way, `String status = ... + ... + ...` as well.